vendor/k8s.io/kubernetes/staging/src/k8s.io/apiserver/pkg/server/config.go

```diff
@@ -403,8 +404,10 @@ func (c completedConfig) New(name string, delegationTarget DelegationTarget) (*G
 
 		listedPathProvider: apiServerHandler,
 
-		swaggerConfig: c.SwaggerConfig,
-		openAPIConfig: c.OpenAPIConfig,
+		swaggerConfig:        c.SwaggerConfig,
+		openAPIConfig:        c.OpenAPIConfig,
+		swaggerAPIContainers: append([]*restful.Container{apiServerHandler.GoRestfulContainer}, delegationTarget.SwaggerAPIContainers()...),
```

Comment:
why do you cache this list in the server? Can't you build up the list when you need it?

Why do we need the list in the first place? Because `apiServerHandler.GoRestfulContainer` can be nil? Would it be enough to have the first non-nil container?